test/e2e/integreatly_test.go

```diff
@@ -168,10 +168,6 @@ func integreatlyMonitoringTest(t *testing.T, f *framework.Framework, ctx *framew
 			if alert.Labels["alertname"] == "KubePodCrashLooping" {
 				continue
 			}
```

Comment:
Ideally I'd like to remove this skip also, I think if we retry a number of times It will also give the `KubePodCrashLooping` alert time to clear.